test/hotspot/jtreg/runtime/Thread/SuspendAtExit.java

```diff
@@ -78,6 +78,8 @@ public static void main(String[] args) {
                     usage();
             }
         }
+        timeMax /= 2;  // Split time between the two sub-tests.
```

Comment:
Same thing about DEF_TIME_MAX.

Reply:
I try not to have the default config for a stress test run longer that 30 seconds total.
15 seconds per sub-test is fine.